.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+
+main: &main
+    machine: true
+    environment:
+      # Used by qthelpers to close widgets after a defined time
+      - TEST_CI: "True"
+      - TEST_CI_APP: "True"
+    steps:
+      - checkout
+      - run:
+          command: docker pull dorowu/ubuntu-desktop-lxde-vnc:trusty
+      - run:
+          name: Install system packages
+          command: |
+            sudo apt-get update
+            sudo apt-get install libegl1-mesa
+      - run:
+          name: Install Miniconda
+          command: |
+            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+            export PATH="$HOME/miniconda/bin:$PATH"
+            conda install python=$PYTHON_VERSION
+      - run:
+          name: Install Spyder from GitHub
+          command: |
+            export PATH="$HOME/miniconda/bin:$PATH"
+            conda install --only-deps spyder
+            mkdir spyder-source
+            cd spyder-source
+            wget -q https://github.com/spyder-ide/spyder/archive/3.x.zip
+            unzip -q 3.x.zip
+            cd spyder-3.x
+            python setup.py install
+      - run:
+          name: Set up test environment
+          command: |
+            export PATH="$HOME/miniconda/bin:$PATH"
+            conda install -q notebook pytest pytest-cov flaky
+            pip install coveralls pytest-qt
+            python setup.py develop
+            echo '********** output of conda list **********'
+            conda list
+      - run:
+          name: Run tests
+          command: |
+            export PATH="$HOME/miniconda/bin:$PATH"
+            mkdir test-reports
+            pytest spyder_notebook --cov=spyder_notebook --junitxml=test-reports/junit.xml
+            COVERALLS_REPO_TOKEN=Kr503QwklmJYKXYRXLywrtw8zbX7K8SKx coveralls
```

Comment:
There's no need to use COVERALLS_REPO_TOKEN in Circle now.

Reply:
Without the token, I get the following error:
```
Couldn't find a repository matching this job.
Traceback (most recent call last):
  File "/home/circleci/miniconda/bin/coveralls", line 11, in <module>
    sys.exit(main())
  File "/home/circleci/miniconda/lib/python3.6/site-packages/coveralls/cli.py", line 80, in main
    log.info(result['url'])
KeyError: 'url'
Exited with code 1
```